src/components/dropdownSearch/MultiSelectDropdownSearch/tests/SelectedOption.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import { mount, ReactWrapper } from 'enzyme';
+// tslint:disable-next-line:no-unused-variable
+import * as React from 'react';
+import { UUID } from '../../../../utils/UUID';
+import {ISelectedOptionProps, SelectedOption} from '../SelectedOption';
+
+describe('SelectedOption', () => {
+  const key: string = UUID.generate();
+  const props: ISelectedOptionProps = {
+    key,
+    displayValue: 'test',
+  };
+
+  describe('<SelectedOption />', () => {
+    let selectedOption: ReactWrapper<ISelectedOptionProps, any>;
+    let selectedOptionInstance: SelectedOption;
+
+    const renderSelectedOption = (props?: ISelectedOptionProps) => {
+      selectedOption = mount(
+        <SelectedOption {...props} />,
+        { attachTo: document.getElementById('App') },
+      );
+      selectedOptionInstance = selectedOption.instance() as SelectedOption;
```

Comment:
It is used to test the instance's methods, if you're not testing any, you can remove it.